src/raftstore/store/fsm/store.rs

```diff
@@ -804,14 +816,11 @@ impl<T, C> RaftPollerBuilder<T, C> {
         }
 
         info!(
-            "[store {}] starts with {} regions, including {} tombstones, {} applying \
-             regions and {} merging regions, takes {:?}",
-            store_id,
-            total_count,
-            tomebstone_count,
-            applying_count,
-            merging_count,
-            t.elapsed()
+            "starts with {} regions, including {} tombstones, {} applying \
+             regions and {} merging regions",
+            total_count, tomebstone_count, applying_count, merging_count;
```

Comment:
why they are not fields?

Reply:
In all the logs, they only appear once, I think it is ok to be part of the msg.

Reply:
I think maybe still better to make it output in fields, so that in the future it will be easy to classify these logs (when log message bodies are fixed).